### ISSUE TITLE
Timeout Enhancement

### ISSUE THREAD

**Issue by alcmoe** [minecraftgenius76/Spleef#2]
Need to teleport Players to another level,when finished building arean,teleport players back again

**Comment by minecraftgenius76** [comment]
Yeah, I know, this need optimization. I found a significant performance different running pocketmine in windows and Linux. complete rebuild of arena is 200% faster than Windows when perform arena.  for normal play this is not an issue because rebuild is floor only.
